### PR TITLE
Fix fakes load

### PR DESCRIPTION
**Pull request by mdbenjam**
We're currently trying to load fakes before setting the application.

**Test Plan**
- [x] Load Reader, then try reloading. Make sure we still get a case.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class ApplicationController < ActionController::Base
   before_action :check_out_of_service
   before_action :strict_transport_security
 
+  before_action :set_application
```
Comment (joofsh) [inline app/controllers/application_controller.rb]: damn, yea you're right I broke this :( thanks for catching & fixing it!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -119,6 +120,10 @@ def setup_fakes
     Fakes::Initializer.setup!(Rails.env, app_name: application)
   end
 
+  def set_application
+    RequestStore.store[:application] = "not-set"
```
Comment (joofsh) [inline app/controllers/application_controller.rb]: What do you think about raising an error or logging a warning here? That way we can force people with new controllers to set their application. It naturally helps us get better prometheus metrics, and helps here with not re-loading fakes in dev mode

Comment (mdbenjam) [inline app/controllers/application_controller.rb]: I'm going to do that in a follow up PR.

Comment (mdbenjam) [inline app/controllers/application_controller.rb]: I'm going to do that in a follow up PR.

**Pull request merged**